papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -123,8 +123,11 @@
                 .map(Availability::getTimeAvailable)
                 .reduce(Duration.ZERO, Duration::plus);
 
-        //to be expanded in issue IO2019TEAM-100
-        double coefficient = sprint.getTimePlanned().dividedBy(sprint.getTimeBurned());
```

Comment:
A `dividedBy` zwraca longa w ogóle, więc trochę zaklinaliśmy rzeczywistość tym `double`

Reply:
ja się pytam gdzie są do tego testy integracyjne

Reply:
:(

Miałam zrobić issue w sumie i może nawet wziąć, ale potem się okazało, że jak pięć osób na raz do mnie pisze nieustannie przez trzy dni przed deadlinem, to nie ma kiedy

No zrobięę, no. Coś jeszcze oprócz sprintów się wydaje testowalne interfacyjnie?

Reply:
Plus ten kod w sumie powstał w okolicy niedzieli, więc i tak bałabym się go opóźniać sugestią "ej, dorzuć testy"

Reply:
jak dla mnie testowalny integracyjnie jest każdy endpoint, trzeba przemyśleć na które testy są potrzebne najbardziej